replication/util/src/main/java/io/deephaven/replication/ReplicatePrimitiveCode.java

```diff
@@ -475,17 +499,9 @@ public static String replaceAll(String sourceClassJavaPath, Map<String, Long> se
         }
         inputStream.close();
 
-        final String sourceClassName = className(sourceClassJavaPath);
-        final String packageName = packageName(sourceClassJavaPath);
-
-        final String resultClassName = replaceAllInternal(sourceClassName, null, exemptions, pairs);
-        final String fullResultClassName = packageName + '.' + resultClassName;
-        final Long serialVersionUID = serialVersionUIDs == null ? null : serialVersionUIDs.get(fullResultClassName);
-        final String resultClassJavaPath = basePath(sourceClassJavaPath) + '/' + resultClassName + ".java";
-        final Map<String, List<String>> noReplicateParts = findNoLocateRegions(resultClassJavaPath);
-
         System.out.println("Generating java file " + resultClassJavaPath);
         String body = replaceAllInternal(inputText.toString(), serialVersionUID, exemptions, pairs);
+        final Map<String, List<String>> noReplicateParts = findNoLocateRegions(resultClassJavaPath);
```

Comment:
I assume you have run *all* of the replicators, including the reflective ones, and ensured that we get no unexpected changes?

Reply:
Yup, tested (and now reran) all reflective and static replicators in replication/reflective and replication/static folders.